### PR TITLE
Jira Integration

### LINKED ISSUE #16: Implement JrTickets

### PR DESCRIPTION
**Pull request by dukris**
@h1alexbel take a look, please
Closes #16

---

### Detailed summary

This PR focuses on adding license information and updating package-info files. The notable changes include:

- Added license information in `LICENSE.txt` and `src/main/java/com/solvd/tickets/package-info.java`
- Updated package-info files in `src/main/java/com/solvd/tickets/tickets`, `src/main/java/com/solvd/tickets/jobs`, and `src/main/java/com/solvd/tickets/source/jira`

> The following files were skipped due to too many changes: `src/main/java/com/solvd/tickets/TicketsApplication.java`, `src/main/java/com/solvd/tickets/source/jira/JiraTicket.java`, `src/main/java/com/solvd/tickets/source/jira/JiraProperty.java`, `src/main/java/com/solvd/tickets/source/jira/JiraConfig.java`, `src/main/java/com/solvd/tickets/jobs/RetrieveTickets.java`, `src/main/java/com/solvd/tickets/source/jira/JiraTickets.java`, `pom.xml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/main/java/com/solvd/tickets/jobs/RetrieveTickets.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+   * Tickets.
+   */
+  private final Tickets tickets;
+
+  @Override
+  @Scheduled(fixedDelayString = "${application.delay}")
+  public void schedule() {
+    List<Ticket> issues = this.tickets.byDate(LocalDate.now());
+    /*
+     * @todo #16:60min/DEV send issues to Harvest.
```
Comment (h1alexbel) [inline src/main/java/com/solvd/tickets/jobs/RetrieveTickets.java:46]: consider more descriptive puzzle text

**Review by h1alexbel — COMMENTED** [review]
@h1alexbel see below

File: src/main/java/com/solvd/tickets/jobs/RetrieveTickets.java [hunk 2]
```diff
@@ -43,7 +43,8 @@ public class RetrieveTickets implements Job {
   public void schedule() {
     List<Ticket> issues = this.tickets.byDate(LocalDate.now());
     /*
-     * @todo #16:60min/DEV send issues to Harvest.
+     * @todo #16:60min/DEV export tickets to Harvest.
+     *   We need to send the tickets to Harvest using appropriate API.
```
Comment (h1alexbel) [inline src/main/java/com/solvd/tickets/jobs/RetrieveTickets.java:47]: specify that we need to do it inside other object

**Review by h1alexbel — CHANGES_REQUESTED** [review]
@hizmailovich see below

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
@rultor merge

**Comment by hizmailovich** [comment]
@rultor merge

**Comment by hizmailovich** [comment]
@rultor merge

File: pom.xml [hunk 3]
```diff
@@ -118,6 +119,10 @@ SOFTWARE.
     </plugins>
   </build>
   <repositories>
+    <repository>
```
Comment (h1alexbel) [inline pom.xml:121]: I think we should remove this repository and try to include jira repository

**Review by h1alexbel — COMMENTED** [review]
@hizmailovich check one

File: pom.xml (hunk 3 above)
Comment (h1alexbel) [reply pom.xml:121]:
```xml
<repositories>
    <repository>
      <id>atlassian-public</id>
      <url>https://packages.atlassian.com/maven/repository/public</url>
    </repository>
  </repositories>
```

**Comment by h1alexbel** [comment]
@hizmailovich I've checked it locally, should work

**Comment by hizmailovich** [comment]
It works locally, yes

**Comment by h1alexbel** [comment]
@hizmailovich GHA also pass, so I think the problem with rultor itself, we try to use version of yegor's image

**Comment by hizmailovich** [comment]
I will change image for rultor

**Comment by h1alexbel** [comment]
@hizmailovich do it in a separate pr

**Comment by hizmailovich** [comment]
Already updated

**Comment by hizmailovich** [comment]
@rultor merge

**Comment by hizmailovich** [comment]
I think it will be better to replace jira sdk with simple http requests to the api at all
@h1alexbel what do you think?

**Comment by h1alexbel** [comment]
@hizmailovich let's merge it, and create a ticket for rultor configuration

**Comment by h1alexbel** [comment]
@hizmailovich let's create puzzle for now

**Pull request merged**